en.json

```diff
@@ -319,7 +364,9 @@
                     "reserved": "Note: This field is reserved for the future."
                 },
                 "theme_shop": {
-                    "title": "Theme Shop"
+                    "title": "Theme Shop",
+                    "under_construction_warning": "This section is under construction.",
```

Comment:
This string shouldn't be translated.

Reply:
> This string shouldn't be translated.

Should [these 2 strings](https://github.com/revoltchat/translations/pull/21/files/3426cc9528d31bd3411b09af3167ec9771e44440#diff-6f6ea5e5e2c2676af6d6cbd64866a01813f39bf2632e0f893970fd774a7d4334R197-R198) (line 197-198) also be removed? I think they are both temporary strings like this one

Reply:
As far as I can tell, the 2FA strings should be okay to keep as 2FA is longer-term - in comparison, the theme shop may be out of beta a lot quicker